test/e2e/resume-parser.spec.ts

```diff
@@ -1,3 +1,101 @@
+import { expect } from 'chai';
+import fs from 'fs';
+import { Education, Experience, HonorOrAward, LicenseOrCertification, Resume } from 'src/types';
+import { convertResumeToJSON } from '../../src';
+
 describe('Resume Parser', () => {
-  it('');
+  const inputPath = 'test/resources/johndoe.html';
+  const outputPath = 'tmp/johndoe.json';
+  let output: Resume;
+
+  before(async () => {
+    await convertResumeToJSON(inputPath, outputPath);
+    output = JSON.parse(fs.readFileSync(outputPath, 'utf8'));
+  });
+
+  it('should contains all the expected keys', async () => {
+    expect(Object.keys(output)).to.deep.equal([
+      'profile',
+      'summary',
+      'experiences',
+      'education',
+      'licensesAndCertifications',
+      'skills',
+      'honorsAndAwards',
+    ]);
+  });
+
+  it('should parse profile correctly', async () => {
```

Comment:
One question in here...I know u are using a Promise on BeforeeEach but is this async needed in here if not *await*  inside this spec? And the same for the rest